source/classes.tex

```diff
@@ -2118,10 +2136,25 @@
 same class as the \grammarterm{nested-name-specifier}.
 \end{itemize}
 
-A destructor shall take no arguments\iref{dcl.fct}.
+A prospective destructor shall take no arguments\iref{dcl.fct}.
 Each \grammarterm{decl-specifier} of the \grammarterm{decl-specifier-seq}
-of a destructor declaration (if any) shall be \tcode{friend}, \tcode{inline}, or
-\tcode{virtual}.
+of a prospective destructor declaration (if any)
+shall be \tcode{friend}, \tcode{inline}, or \tcode{virtual}.
+
+\pnum
+At the end of the definition of a class,
+overload resolution is performed
+among the prospective destructors declared in that class
+with an empty argument list
+to select the \defn{destructor} for the class,
+also known as the \defnadj{selected}{destructor}.
+The program is ill-formed if overload resolution fails.
+Destructor selection does not constitute
+a reference to,
+or odr-use\iref{basic.def.odr} of,
```

Comment:
If we add the reference after "reference to" as suggested in the paper, it would read better if the reference here were moved after the "of".

Reply:
I intentionally moved the reference to [dcl.fct.def.delete] to the end of the sentence, so it more clearly attaches to its referent.